tests/drivers/fuel_gauge/sbs_gauge/src/test_sbs_gauge.c

```diff
@@ -134,23 +134,66 @@ ZTEST_USER_F(sbs_gauge_new_api, test_set_some_props_failed_returns_failed_prop_c
 ZTEST_USER_F(sbs_gauge_new_api, test_set_prop_can_be_get)
 {
 	uint16_t word = BIT(15) | BIT(0);
-	struct fuel_gauge_set_property set_prop = {
-		/* Valid property */
-		.property_type = FUEL_GAUGE_SBS_MFR_ACCESS,
-		/* Set Manufacturer's Access to 16 bit word*/
-		.value.sbs_mfr_access_word = word,
+	struct fuel_gauge_set_property set_props[] = {
+		{
+			/* Valid property */
```

Comment:
Was this a `clang-format` change?

Reply:
No actually, earlier it's single property, now it's list of properties, hence the indentation is added